frontend/src/components/Sidepanel/Panels/TestingLab/TestingLab.tsx

```diff
@@ -248,9 +310,61 @@ const TestingLab = () => {
     }
 
     // Add 'REJECTED'/'ACCEPTED' label
-    return `${transitionsWithRejected.join('\n')}${(traceIdx === lastTraceIdx) ? '\n\n' + (accepted ? 'ACCEPTED' : 'REJECTED') : ''}`
-  }, [traceInput, simulationResult, statePrefix, traceIdx, getStateName])
+    return `${transitionsWithRejected.join('\n')}${(traceIdx === lastTraceIdx || (enableManualStepping && currentManualSuccessors.length === 0)) ? '\n\n' + (accepted ? 'ACCEPTED' : 'REJECTED') : ''}`
+  }, [traceInput, simulationResult, statePrefix, traceIdx, getStateName, enableManualStepping])
 
+  // Creates problem and sets it, should run every time trace/graph changes while enableManualStepping is enabled
+  useEffect(() => {
+    if (enableManualStepping) {
+      const _problem = buildProblem(graph, traceInput)
+      if (_problem != null) {
+        console.log('currentManualNode reset')
```

Comment:
You may want to comment out the console.log() calls to make it a bit cleaner?
```suggestion
    
```

Reply:
Yeah, forgot to remove those